p2p/transport/webtransport/conn.go

```diff
@@ -60,7 +60,18 @@ func (c *conn) AcceptStream() (network.MuxedStream, error) {
 	return &stream{str}, nil
 }
 
-func (c *conn) Close() error             { return c.session.Close() }
+func (c *conn) allowWindowIncrease(size uint64) bool {
+	return c.scope.ReserveMemory(int(size), network.ReservationPriorityMedium) == nil
```

Comment:
What's the argument for Medium vs High? Medium bounds us to 60% of available memory. So a 4GiB node with defaults would have 0.6(512+128MiB)=384MiB here. I think that sounds fine, just want to check though

Reply:
When we're running low on memory, we want to block allocations that are not strictly necessary. Performance will be slightly degraded if you don't get the full receive window, but things will still work. Whereas if you fail to allocate the buffer needed for parsing a protobuf (for example), things will break in a lot worse ways.